simple_use_case/source/poisson.py

```diff
@@ -59,28 +60,39 @@ def solve_and_write_output(mesh: str, degree: int, outputfile: str):
         Degree of the finite element space.
     outputfile : str
         FilePath to the output file into which the solution is written.
+    numdofs : str
+        FilePath to which the number of DoFs is written.
     """
     discrete_solution = solve_poisson(mesh, degree)
     discrete_solution.rename("u", discrete_solution.name())
     resultFile = df.File(outputfile)
     resultFile << discrete_solution
 
+    with open(numdofs, "w") as handle:
+        dofs = discrete_solution.function_space().dim()
+        handle.write("{}\n".format(dofs))
+
```

Comment:
```suggestion
    dofs = discrete_solution.function_space().dim()
    with open(numdofs, "w") as handle:
        handle.write("{}\n".format(dofs))
    print(f"Number of dofs used: {dofs}")
```

This way, tools that can parse values without the detour over files (e.g. cwl by grepping stdout) don't have to, right?

Reply:
Yes. `doit` can parse values without the detour over files by the way. But I would keep the additional text file output if one of the tools might not support grepping stdout directly.

Reply:
yeah, I agree. That's why I left it in the suggestion.